README.md

```diff
@@ -18,6 +18,11 @@ Please use appropriate deployment yaml files available here - <https://github.co
 
 Note: `v1.0.2`, deployment yamls files are compatible with `v1.0.1`.
 
+### v2.0.1
+
+* gcr.io/cloud-provider-vsphere/csi/release/driver:v2.0.1
+* gcr.io/cloud-provider-vsphere/csi/release/syncer:v2.0.1
+
 ### v2.0.0
```

Comment:
Can we put a quick note here that` v2.0.0` is not compatible with the vSphere 67u3 release?

Reply:
Can you explain more on what do you mean by that? Are you referring to a known issue in the syncer?

Reply:
Basically wanted to alert users not to use `v2.0.0` on vSphere 67u3 release directly from the landing page, if they have not read the documentation and not read about this from known issue page.

Reply:
Added to notes on this page - "It is recommended to use `v2.0.1` if you're on `vSphere 6.7 Update3`".